docs/logging-usage.rst

```diff
@@ -314,3 +314,36 @@ Delete a sink:
    >>> sink.delete()  # API call
    >>> sink.exists()  # API call
    False
+
+Python Logging Integration:
+
+It's possible to tie the Python :mod`logging` module directly into Google Cloud Logging. To use it,
+create a :class:`CloudLoggingHandler <gcloud.logging.CloudLoggingHandler` instance from your Logging client.
+
+.. doctest::
+    >>> import logging
+    >>> import gcloud.logging # Don't conflict with standard logging
+    >>> from gcloud.logging.handlers import CloudLoggingHandler
+    >>> client = logging.Client()
+    >>> handler = CloudLoggingHandler(client)
+    >>> cloud_logger = logging.getLogger('cloudLogger')
+    >>> cloud_logger.setLevel(logging.INFO) # defaults to WARN
+    >>> cloud_logger.addHandler(handler)
+    >>> cloud_logger.error('bad news') # API call
+
+.. note: This handler currently only supports a synchronous API call, which means each logging statement that uses this handler will require an API call.
```

Comment:
done, I had a bunch of mistakes because I wasn't looking more closely at the generated docs, looks closer to what we want now